lib/paper_trail.rb

```diff
@@ -126,9 +128,9 @@ def self.paper_trail_store
 
   # Returns PaperTrail's configuration object.
   def self.config
-    @@config ||= PaperTrail::Config.instance
-    yield @@config if block_given?
-    @@config
+    @config ||= PaperTrail::Config.instance
```

Comment:
Was there a pro / con to storing this as an instance or class level variable? Is this being done to address thread safety concerns?